npm/vue2/cypress.json

```diff
@@ -1,9 +0,0 @@
-{
```

Comment:
Are we sure we don't want to retain at least some tests over this module? I agree it makes sense not to duplicate everything (especially the examples), but deleting the entire test setup feels a bit much. This is still going to be published as a separate package on NPM that we have to maintain, and I'm not sure we have test coverage of it other than the deleted files.